bin/ci-tests.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -eu
+
+[ "$SKIP_TESTS" = 'true' ] && echo "Skipping tests" && exit 0
+
+cp -r .demo/ env/
```

Comment:
does `.demo` still exist?

Reply:
The `.demo` does not exist but the `bin/ci-tests.sh` should be removed as it not used anymore.